packages/slate/test/models/change/with-mutations-no-option-override.js

```diff
@@ -0,0 +1,69 @@
+/** @jsx h */
+
+import h from '../../helpers/h'
+
+export const schema = {
+  blocks: {
+    'paragraph': {},
+    'list-item': {
+      parent: { types: ['unordered-list'] },
+      nodes: [
+        { objects: ['text'] }
+      ]
+    },
+    'unordered-list': {},
+  }
+}
+
+export const customChange = (change) => {
+  // see if we can break the expected validation sequence by toggling
+  // the normalization option
+  let target = change.value.document.nodes.get(0)
+  change.wrapBlockByKey(target.key, 'list-item', { normalize: true })
+  target = change.value.document.nodes.get(0)
+  change.wrapBlockByKey(target.key, 'unordered-list', { normalize: false })
+}
+
+export const input = (
+  <value>
+    <document>
+      <paragraph />
+    </document>
+  </value>
+)
+
+export const output = {
```

Comment:
Any specific reason we're using the JSON version of the output here? It might be nicer to write tests using the JSX version instead (like the `./test/changes` tests do) for terseness if we can.

Reply:
It was what was familiar, happy to change it to the other style

Reply:
@CameronAckermanSEL cool, yeah let's change it. It'll make adding future tests easier I think too.